### PR TITLE
feat - add home page

### PR DESCRIPTION
**Pull request by mcmonojit**
**Added home page**

Preview:  https://reverent-euler-6d0946.netlify.app/

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+
+@import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@400;500;600;700&display=swap');
+
+/* font - poppins  */
+
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700;800&display=swap');
+
+
+@import url(https://complib.netlify.app/css/allcomponent.css);
+
```
Comment (Sharmaryan) [inline style.css:10]: too much white space

File: index.html [hunk 2]
```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (Sharmaryan) [inline index.html:6]: you can add meta tag for author , although it is not mandatory, just a suggestion

File: index.html [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+    <title>Barfi</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="navbar-container">
+        <nav class="navbar">
+          <a href="/"><div class="navbar-brand-name">Barfi</div></a>
```
Comment (Sharmaryan) [inline index.html]: you can add that anchor tag inside div

**Review by Sharmaryan — APPROVED** [review]
suggested some changes, you can make them if you want otherwise everything is fine

- UI looks clean
- naming convention is also good
- you can put that color variables css file in separate folder

**Comment by mcmonojit** [comment]
Thanks @Sharmaryan , I have applied the recommended changes in [529ab6a](https://github.com/mcmonojit/Barfi/pull/1/commits/529ab6a099af02be6916902d3e533d20b01051b5).

File: css/colorVariables.css [hunk 4]
```diff
@@ -0,0 +1,40 @@
+:root {
+    --navyBlue: #082032;
```
Comment (bharati-21) [inline css/colorVariables.css:2]: I suggest not to include the variable value in the variable name.

Reason - If in future you decide to change navyBlue to say purple, you'd have to change the references to this variable wherever you used. 
Please choose a generic enough name.

File: css/colorVariables.css [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+    --lightShadeOfBlack: #303030;
+    --darkBlack: #000;
+    --lightWaterBlue: #194975;
+    --lightShadeOfGrey: #e5e5e5;
+
+    --checkSuccess: #16a34a;
+    --checkWarning: #d97706;
+    --checkError:#ef4444;
+}
```
Comment (bharati-21) [inline css/colorVariables.css:25]: The above suggestion for all the colors.

File: index.html (hunk 2 above)
Comment (bharati-21) [reply index.html:6]: Similar suggestion. This increases the accessibility of your website and helps in SEO.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+      <div class="footer-about">
+        <div class="newsletter">
+          <p>Hear from us regularly!</p>
+          <div class="newsletter-input">
+            <input placeholder="example@email.com">
+            <button>Subscribe</button>
+          </div>
+        </div>
+
+        <p class="copyright">Copyright &copy; <script>document.write(new Date().getFullYear())</script> Barfi</p>
```
Comment (bharati-21) [inline index.html:133]: Nice use of dynamic date!

File: style.css [hunk 7]
```diff
@@ -0,0 +1,267 @@
...
+    height: 20px;
+}
+
+.social-icons{
+    width: 12rem;
+    margin: 1rem;
+    display: flex;
+    justify-content: space-evenly;
+}
+
```
Comment (bharati-21) [inline style.css:269]: I see there's heavy usage of flex properties. A suggestion you can work on if you have the bandwidth right now - create util classes for flex.

**Review by bharati-21 — APPROVED** [review]
Dropped a few suggestions. I really liked the usage of dynamic year! 

You can look into them. But apart from them, all looks good. 👍🏽

**Comment by mcmonojit** [comment]
Thanks @bharati-21 for the review. I have applied some of the recommended changes. I will apply the naming of color variables in a separate PR.

**Pull request merged**